src/Persistence.Tests/_TestData/ValidYaml-CI/BuiltInControl/with-template-name.pa.yaml

```diff
@@ -1,4 +1,4 @@
 button with template name:
-  Control: Button 
+  Classic: Button 
```

Comment:
we don't have `Classic:` in the schema. Let's discuss during standup

Reply:
I don't think the goal is to have every possible property in the schema. For example, we don't have Width, Hight, X, Y, etc

Reply:
Width, Height, X, Y are not in the schema. correct. they are extension points under the 'Properties' bag.
But this property is very much in the schema.

As to the merits of this proposal, this sort of thing was explicitly rejected in the schema discussions when we had a similar proposal for Component instances.

For context, the proposal looked something like this: 
For a Component defined in the local app with a name of `MyHeader`, a control instance would look like:
```yaml
- MyHeader_1:
    Component: MyHeader
    Properties:
      Title: ="My cool app"
      X: =0
      Y: =0
      Width: =Parent.Width
```

This was rejected by this team and architects for the reason of perceived confusion and complexity for users. Namely, that the control instance doesn't look like all other control instances.
Note, a Component instance is definitely a 1st class construct, which I believe warrants a custom implementation. Yet conceded due to overwhelming concerns.

So we kept it looking like:
```yaml
- MyHeader_1:
    Control: Component
    ComponentName: MyHeader
    Properties:
      Title: ="My cool app"
      X: =0
      Y: =0
      Width: =Parent.Width
```

Now, for classic vs modern control instances, there isn't a 1st class control type. We're only trying to resolve a unique invariant name for the control type. So I would not suggest making these controls use a new schema keyword for it just because we've got some naming conflicts.

I feel like we should have this discussion with McCall and other doc server team members in order to have a plan.

Reply:
I understand the discussion about components but this PR is about PCF/non-PCF controls